docs/google/calibration.md

```diff
@@ -0,0 +1,217 @@
+# Calibration Metrics
+
+Processors that are quantum computers periodically undergo calibrations to
```

Comment:
Simplify by replacing "Processors that are quantum computers" with "Quantum processors".

Reply:
Done.